submissions/Barasii/a-tiny-JS-world-OOP/index.js

```diff
@@ -0,0 +1,86 @@
+class Inhabitant {
+   constructor(species, name, gender, legs, hands, saying, friends) {
+      this.species = species;
+      this.name = name;
+      this.gender = gender;
+      this.legs = legs;
+      this.hands = hands;
```

Comment:
- [ ] Classes for species that do not have hands by natural design do not consequently have `hands` or any equivalent property
- [ ] OOP, SOLID and DRY principles are intensively employed

Every property from base class is inherited by a child class. This makes all your classes have `hands`, `legs` etc

Image you add fish, bird etc 
Your base class becomes cluttered and therefore breaks OOP and SOLID principles.

`class B extends A` - "extends" is a keyword here. Recap inheritance principle from OOP and fix the code accordingly.

Reply:
ThanksI removed